src/hooks/use-debug-tap-listener.ts

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from 'react';
+import { Subject } from 'rxjs';
+import { buffer, debounceTime, filter, map } from 'rxjs/operators';
+
+import { ScreensEnum } from '../navigator/enums/screens.enum';
+import { useNavigation } from '../navigator/hooks/use-navigation.hook';
+
+const TAPS_COUNT = 10;
+const DEBOUNCE_TIME = 500;
+
+export function useDebugTapListener() {
```

Comment:
A component with `View` as root element and children seems to be designed for non-touchable elements as children

Reply:
I have tried, and it works with `View` (have tested on both platforms)

Reply:
Really? Did you try pressing exactly 10 times from 'Wallet' page?

Reply:
yep, firstly Settings page is opened (on the first press), and after this Debug is dispalyed

Reply:
You may be wondered but I need 11 taps
Sorry, I can't shoot a video because I couldn't find an option to show a circle when and where a screen is touched